src/calculateRentalCost.js

```diff
@@ -29,6 +29,22 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const dailyPrice = 40;
+  const longTermPeriod = 7;
+  const longTermDiscount = 50;
+  const shortTermPeriod = 3;
+  const shortTermDiscount = 20;
+  let totalCost = dailyPrice * days;
```

Comment:
```suggestion
  const totalCost = dailyPrice * days;
```
it's better to use constant here and in any place you need just `return totalCost` or `(totalCost - someTermDiscount)`. Don't forget brackets around expressions.